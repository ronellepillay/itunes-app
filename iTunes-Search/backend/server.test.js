const request = require('supertest');
const app = require('./server');

//Snapshot
describe('GET /favourites', () => {
  it('returns a snapshot of the favourites list', async () => {
    const res = await request(app)
      .get('/favourites')
      .expect('Content-Type', /json/)
      .expect(200);

    expect(res.body).toMatchSnapshot();
  });
});

//Unit test
describe('GET /favourites', () => {
  it('returns a JSON response with the favourites list', async () => {
    const res = await request(app)
      .get('/favourites')
      .expect('Content-Type', /json/)
      .expect(200);

    expect(Array.isArray(res.body)).toBe(true);
  });
});
