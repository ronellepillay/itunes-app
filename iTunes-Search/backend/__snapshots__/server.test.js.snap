// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GET /favourites returns a snapshot of the favourites list 1`] = `[]`;
